src/modules/ComicList/model/Comic.ts

```diff
@@ -0,0 +1,6 @@
+export default interface IComic {
```

Comment:
Is this approach from java?
Let's remove `model` directory and put this interface to modules/ComicList/types.ts

```suggestion
export default interface Comic {
```